tests/unit/amo/components/TestAddonHead.js

```diff
@@ -84,113 +85,50 @@ describe(__filename, () => {
     );
   });
 
-  it('renders a description meta tag containing the add-on summary', () => {
+  it('renders a HeadMetaTags component', () => {
     const addon = createInternalAddon(fakeAddon);
-    const root = render({ addon });
-
-    expect(root.find('meta[name="description"]')).toHaveLength(1);
-    expect(root.find('meta[name="description"]')).toHaveProp(
-      'content',
-      `Download ${addon.name} for Firefox. ${addon.summary}`,
-    );
-  });
-
-  it('renders Open Graph meta tags', () => {
     const lang = 'fr';
-    const addon = createInternalAddon(fakeAddon);
     const { store } = dispatchClientMetadata({ lang });
-    const root = render({ addon, store });
 
-    [
-      ['og:type', 'website'],
-      ['og:url', addon.url],
-      ['og:locale', lang],
-      ['og:image', addon.previews[0].image_url],
-    ].forEach(([property, expectedValue]) => {
-      expect(root.find(`meta[property="${property}"]`)).toHaveProp(
-        'content',
-        expectedValue,
-      );
-    });
+    const root = render({ addon, store });
 
-    expect(root.find(`meta[property="og:title"]`).prop('content')).toContain(
-      addon.name,
+    expect(root.find(HeadMetaTags)).toHaveLength(1);
+    expect(root.find(HeadMetaTags)).toHaveProp('date', addon.created);
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'description',
+      `Download ${addon.name} for Firefox. ${addon.summary}`,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'image',
+      addon.previews[0].image_url,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'lastModified',
+      addon.last_updated,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'title',
+      `${addon.name} – Get this Extension for 🦊 Firefox Android (${lang})`,
```

Comment:
Why `Android`? If that is expected should the test call `dispatchClientMetadata` with `clientApp: CLIENT_APP_ANDROID`?

Reply:
We use `CLIENT_APP_ANDROID` by default in `dispatchClientMetadata()`, but I will add it.